### PR TITLE
Refactor tests to Pest

### PR DESCRIPTION
**Pull request by alexmanase**
wip

### CODE REVIEW

**Comment by rubenvanassche** [comment]
Hi @alexmanase, this is looking great already, let me know when you're finished then I'll take a look!

**Comment by alexmanase** [comment]
Hi @rubenvanassche!

Thanks! I will notice you when I'm done.

File: tests/Datasets/Attributes/RulesDataset.php [hunk 1]
```diff
@@ -0,0 +1,987 @@
...
+) {
+    return [
+        'attribute' => $attribute,
+        'expected' => $expected,
+        'expectedCreatedAttribute' => $expectCreatedAttribute ?? $attribute,
+        'exception' => $exception,
+    ];
+}
+
+dataset('attributes', function () {
```
Comment (alexmanase) [inline tests/Datasets/Attributes/RulesDataset.php:104]: For this dataset, I had to wait to see if a PR is going to be merged(https://github.com/pestphp/pest/pull/602) in order to have a more readable dataset.

**Comment by alexmanase** [comment]
Hi @rubenvanassche! I think you can take a look.

**Pull request merged**